merge_lora.py

```diff
@@ -2,21 +2,26 @@
 import sys
 
 sys.path.append("llm-finetune/lit-gpt")
-sys.path.append("llm-finetune/lit-gpt/script")
+sys.path.append("llm-finetune/lit-gpt/scripts")
 
-import merge_lora as merge
+import scripts.merge_lora as merge
 
-
-def merge_lora(
+def perform_merge(
     model_name: str = "mistralai/Mistral-7B-v0.1",
+    lora_path: str = "out/lora/alpaca/lit_model_lora_finetuned.pth",
+    out_dir: str = "out/lora/checkpoint"
 ):
-    mege.merge_lora(
+    out_dir = Path(out_dir)
+    # Make the out_dir, because the merge_lora script doesn't do that...
+    out_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
I merged https://github.com/Lightning-AI/lit-gpt/pull/829 to fix this. Thank you!